pkg/simple/client/devops/jenkins/pipeline_internal.go

```diff
@@ -795,6 +796,8 @@ func createMultiBranchPipelineConfigXml(projectName string, pipeline *devopsv1al
 		appendGitSourceToEtree(source, pipeline.GitSource)
 	case "github":
 		appendGithubSourceToEtree(source, pipeline.GitHubSource)
+	case "gitlab":
```

Comment:
you've already defined const in https://github.com/kubesphere/kubesphere/pull/3131/files#diff-99571c7b6e87a4eec2b7446a2ee109a8579e7d859e57421ef17729d80c8731fcR80 , why not use them

Reply:
I've fixed it.